### DISCUSSION TITLE
Add support for manual retry of downloading attachments

### DISCUSSION THREAD

**Discussion by hseg** [d99kris/nchat#192] [Ideas]
Came up when testing #191 -- once `nchat` has given up on an attachment, it is marked as unavailable, even if the error was on the client end. Either be smarter about this, or just give a mechanism to force-retry downloading.

**Comment by d99kris** [comment]
Thanks for suggesting an improvement. Can I confirm you're using WhatsApp?
For Telegram there's a manual way (deleting cache files in the filesystem) to clear attachment cache state, but no such way in WhatsApp currently.

**Comment by hseg** [comment]
On Fri, Mar 01, 2024 at 05:34:56AM -0800, Kristofer Berggren wrote:
> Thanks for suggesting an improvement. Can I confirm you're using WhatsApp?
> For Telegram there's a manual way (deleting cache files in the filesystem) to clear attachment cache state, but no such way in WhatsApp currently.

Yes, I'm using Whatsapp, sorry for not specifying.

**Reply by d99kris** [reply]
No worries, thanks for confirming 👍
